### ISSUE TITLE
[Keyboard navigation - Make code - Share project]: Keyboard focus does not land on the revealed content by invoking the 'Embed code' button. 

### ISSUE THREAD

**Issue by vroopea** [microsoft/pxt-microbit#5487]
### User Experience:

Keyboard and screen reader dependent people will get impacted if keyboard focus does not land on the revealed content by invoking the 'Embed code' button. Due to which the end user needs to make an extra effort to reach the revealed content which is time consuming.

### Note:
User credentials should NOT be included in the bug.

### Repro Steps:

1. Launch application URL : https://makecode.microbit.org/.
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Share" option appearing in header section and select it.
6. Navigate through all the controls of "Share Project" dialog and invoke share project.
7. Navigate to the 'Embed code' button and expand it.
8. Observe whether keyboard focus lands on the revealed content or not.

### Actual Result:
Issue: Keyboard focus does not land on the revealed content by invoking the 'Embed code' button. 

**Observation:**
- By invoking the 'Embed code' button, tab items appears but the keyboard focus does not land on the first interactive element of the revealed content.
- Screen reader remains silent after invoking the 'Embed code' button.

### Expected Result:
Keyboard focus should land on the revealed content by invoking the 'Embed code' button.

**Example:** By invoking the 'Embed code' button, the keyboard focus should land on the first interactive element of the revealed content i.e., on the 'Code' tab item.

### MAS Reference:
[MAS 2.4.3 – Focus Order](https://aka.ms/MAS2.4.3) 

### ARIA Authoring Ref Link/Fluent Patterns Guidance /HITS Guidance:
NA

### UIA/DOM details available:
NA

### Reference Links:

- [Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
- External Bug Process: If this bug belongs to an external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [External Bug Process (sharepoint.com)](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDN8MDF8fDE1MThmM2JlZGFjMDRlZDhiZmViYTUxMDA2M2NiZWI3fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2Mzc3NDEyNTM2MTEzNzIwNzR8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaUlpd2lRVTRpT2lJaUxDSlhWQ0k2TVRKOQ%3D%3D&sdata=MnlpSXlVMnpGT3cvZ0xJNnFWUWZWTGt6dmo1VkJMZFpIZHBvUUduNDhyMD0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-kgeetala%40microsoft.com). “HCL Staff should not log any third-party external bugs. Should be routed to EDAD team”.
- Please reach out to [C&AI Teams channel](https://apc01.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906844600&sdata=22JjBMoA1ID/KnYqFa/OB%2BrkTaLh5YYsGBd/7Bm7u5w%3D&reserved=0) for any process-related queries.

### Test Environment:

- OS Version: Windows 11 23H2 (OS Build:22631.3007)
- Browser: Chrome Version 121.0.2277.4 (Official build) dev (64-bit).
- Test Matrix: Chrome + JAWS
- URL: https://makecode.microbit.org/.

https://github.com/microsoft/pxt-microbit/assets/158465724/c583817d-0aab-42fd-a7d9-6f818833b799

**Comment by vroopea** [comment]
#Accessibility;#A11yMAS;#A11ySev3;#MAS2.4.3;#AILimited;#HCL;#Benchmark;#Win11-Chrome;#E2E-HCL-MakeCode-Feb2024;
